src/main/scala/diplomacy/Nodes.scala

```diff
@@ -737,6 +1571,7 @@ class MixedNexusNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   protected[diplomacy] def mapParamsU(n: Int, p: Seq[UO]): Seq[UI] = { if (n > 0) { val a = uFn(p); Seq.fill(n)(a) } else Nil }
 }
 
+/** [[NexusNode]] is a [[MixedNexusNode]], in which `inner` and `outer` have the same [[NodeImp]]. */
```

Comment:
`in which the inward and outward side of the node have the same [NodeImp]] implementation.`